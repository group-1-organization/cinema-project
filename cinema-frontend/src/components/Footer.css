.footer {
    padding: 15px;
    text-align: center;
    background: #0e254d;
    color: white;
    /* font-size: 10px; */
    /* position: relative; */
    bottom:0;
    /* padding-top: 20px;
    padding-bottom: 20px; */
    
}