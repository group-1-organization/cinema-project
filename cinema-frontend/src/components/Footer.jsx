import '../App.css'
import { Row } from 'react-bootstrap';
import { Col } from 'react-bootstrap';
import { Link } from 'react-router-dom';

const Footer = () => {
    return (
        <div className="footer">
            <div>
                <Row>
                    <Col>
                        <h4>Where to find us</h4>
                        <p>Bentley Bridge Park</p>
                        <p>Wednesfield Way</p>
                        <p>Wolverhampton</p>
                        <p>WV11 1TZ</p>
                    </Col>
                    <Col>
                        <h4>Find out more</h4>
                        <Link to={'/'}>Home</Link>
                        <br />
                        <Link to={'/aboutus'}>About Us</Link>
                        <br />
                        <Link to={'/contact'}>Contact Us</Link>
                    </Col>
                    <Col>
                        <h4>Follow Us</h4>
                        <a href="https://www.facebook.com/QALtd/">Facebook</a>
                        <br />
                        <a href="https://twitter.com/QA_Ltd">Twitter</a>
                        <br />
                        <a href="https://www.instagram.com/qalimited/?hl=en">Instagram</a>
                        <br />
                        <a href="https://www.youtube.com/channel/UCr5kx6_fYtqx0LLD7aW2zIg">YouTube</a>
                        <br />
                    </Col>
                </Row>
            </div>
            <div>
                <p>QA Cinema &copy; 2021</p>
            </div>
        </div>
    );
}

export default Footer;