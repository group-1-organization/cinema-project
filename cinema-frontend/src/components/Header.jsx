import Nav from 'react-bootstrap/Nav';
import Navbar from 'react-bootstrap/Navbar';
import { Link } from 'react-router-dom';
import Form from 'react-bootstrap/Form';
import Button from 'react-bootstrap/Button';
import FormControl from 'react-bootstrap/FormControl';
import Image from 'react-bootstrap/Image';
import NavDropdown from 'react-bootstrap/NavDropdown';

const Header = () => {
    return (
        <Navbar collapseOnSelect expand="lg" bg="dark" variant="dark">
            <Link to={"/"} className="nav-link"><Image src="QA.png" width="50" /> </Link>
            <Navbar.Toggle aria-controls="responsive-navbar-nav" />
            <Navbar.Collapse id="responsive-navbar-nav">
                <Nav className="mr-auto">
                    <Nav.Item>
                        <Link to={"/listing"} className="nav-link">Listings</Link>
                    </Nav.Item>
                    <Nav.Item>
                        <Link to={"/upcoming"} className="nav-link">Upcoming</Link>
                    </Nav.Item>

                    <Nav.Item>
                        <Link to={"/bookings"} className="nav-link">Bookings</Link>
                    </Nav.Item>
                    <NavDropdown title="Info" id="navbarScrollingDropdown" >
                        <NavDropdown.Item><Link to={"/gettingthere"}>Getting There</Link></NavDropdown.Item>
                        <NavDropdown.Divider />
                        <NavDropdown.Item ><Link to={"/placestogo"}>Places to go</Link></NavDropdown.Item>
                        <NavDropdown.Divider />
                        <NavDropdown.Item ><Link to={"/openingtimes"}>Opening Times</Link></NavDropdown.Item>
                        <NavDropdown.Divider />
                        <NavDropdown.Item ><Link to={"/classifications"}>Classifications</Link></NavDropdown.Item>
                    </NavDropdown>
                    <Nav.Item>
                        <Link to={"/discussion"} className="nav-link">Discussion Board</Link>
                    </Nav.Item>
                </Nav>
                <Form inline>
                    <FormControl type="text" placeholder="Search" className="mr-sm-2" />
                    <Button variant="outline-success">Search</Button>
                </Form>
            </Navbar.Collapse>
        </Navbar>
    )
}

export default Header