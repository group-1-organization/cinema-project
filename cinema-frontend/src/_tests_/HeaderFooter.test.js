import renderer, { act } from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import Header from '../components/Header';
import Footer from '../components/Footer';

test('Header matches snapshot', () => {
    const tree = renderer.create(<Router><Header /></Router>);

    expect(tree).toMatchSnapshot();
});

test('Footer matches snapshot', () => {
    const tree = renderer.create(<Router><Footer /></Router>);

    expect(tree).toMatchSnapshot();
});