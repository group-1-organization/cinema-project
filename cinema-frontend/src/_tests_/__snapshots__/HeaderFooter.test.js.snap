// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer matches snapshot 1`] = `
<div
  className="footer"
>
  <div
    className="mb-0"
  >
    <div
      className="row"
    >
      <div
        className="col-sm-4"
      >
        <h4>
          Where to find us
        </h4>
        <ul
          className="list-unstyled"
        >
          <li>
            Bentley Bridge Park
          </li>
          <li>
            Wednesfield Way
          </li>
          <li>
            Wolverhampton
          </li>
          <li>
            WV11 1TZ
          </li>
        </ul>
      </div>
      <div
        className="col-sm-4"
      >
        <h4>
          Find out more
        </h4>
        <a
          href="/"
          onClick={[Function]}
        >
          Home
        </a>
        <br />
        <a
          href="/aboutus"
          onClick={[Function]}
        >
          About Us
        </a>
        <br />
        <a
          href="/contact"
          onClick={[Function]}
        >
          Contact Us
        </a>
      </div>
      <div
        className="col-sm-4"
      >
        <h4>
          Follow Us
        </h4>
        <a
          href="https://www.facebook.com/QALtd/"
        >
          Facebook
        </a>
        <br />
        <a
          href="https://twitter.com/QA_Ltd"
        >
          Twitter
        </a>
        <br />
        <a
          href="https://www.instagram.com/qalimited/?hl=en"
        >
          Instagram
        </a>
        <br />
        <a
          href="https://www.youtube.com/channel/UCr5kx6_fYtqx0LLD7aW2zIg"
        >
          YouTube
        </a>
        <br />
      </div>
    </div>
  </div>
  <div>
    <p>
      QA Cinema Â© 2021
    </p>
  </div>
</div>
`;

exports[`Header matches snapshot 1`] = `
<nav
  className="navbar navbar-expand-lg navbar-dark bg-dark"
>
  <a
    className="nav-link"
    href="/"
    onClick={[Function]}
  >
    <img
      className=""
      src="/images/qa-logo.png"
      width="50"
    />
     
  </a>
  <button
    aria-controls="responsive-navbar-nav"
    aria-label="Toggle navigation"
    className="navbar-toggler collapsed"
    onClick={[Function]}
    type="button"
  >
    <span
      className="navbar-toggler-icon"
    />
  </button>
  <div
    aria-expanded={null}
    className="navbar-collapse collapse"
    id="responsive-navbar-nav"
  >
    <div
      className="mr-auto navbar-nav"
      onKeyDown={[Function]}
    >
      <div
        className="nav-item"
      >
        <a
          className="nav-link"
          href="/listing"
          onClick={[Function]}
        >
          Listings
        </a>
      </div>
      <div
        className="nav-item"
      >
        <a
          className="nav-link"
          href="/upcoming"
          onClick={[Function]}
        >
          Upcoming
        </a>
      </div>
      <div
        className="nav-item"
      >
        <a
          className="nav-link"
          href="/bookings"
          onClick={[Function]}
        >
          Bookings
        </a>
      </div>
      <div
        className="nav-item dropdown"
      >
        <a
          aria-expanded={false}
          aria-haspopup={true}
          className="dropdown-toggle nav-link"
          data-rb-event-key={null}
          href="#"
          id="navbarScrollingDropdown"
          onClick={[Function]}
          onKeyDown={[Function]}
          role="button"
        >
          Info
        </a>
      </div>
      <div
        className="nav-item"
      >
        <a
          className="nav-link"
          href="/discussion"
          onClick={[Function]}
        >
          Discussion Board
        </a>
      </div>
    </div>
    <form
      className="form-inline"
    >
      <input
        className="form-control"
        id="movieSearch"
        onBlur={[Function]}
        onFocus={[Function]}
        onInput={[Function]}
        placeholder="Search Term"
        type="text"
      />
      <a
        aria-label="search-link"
        href="/"
        onClick={[Function]}
      >
        <button
          aria-label="search-btn"
          className="button-colour adj-search btn btn-primary"
          disabled={false}
          id="movieSearchSubmit"
          type="button"
        >
          Search Term
        </button>
      </a>
    </form>
  </div>
</nav>
`;
